### PR TITLE
Updating the Backup and restore doc

### PR DESCRIPTION
**Pull request by dockerymick**
https://app.zenhub.com/workspaces/engineering-backlog-do-not-delete-604fab62d4b98d00150a2854/issues/stolostron/backlog/22748

### CODE REVIEW

**Comment by dockerymick** [comment]
Changes were tracked in this Google doc: https://docs.google.com/document/d/1teUMWW_SsFbYTXWfMfjQEjJH2Y893Exvpp4WNacecFg/edit#

File: clusters/backup_and_restore.adoc [hunk 1]
```diff
@@ -178,7 +178,12 @@ Resources are separately backed up in the following groups:
...
+*Note*: The _resource backup_ file contains managed cluster-specific resources, but does not contain the subset of resources that connect managed clusters to the hub cluster. The resources that connect managed clusters are also called activation resources and are contained in the managed clusters backup. When you restore backups only for the _credentials_ and _resource_ backup on a new hub cluster, the new hub cluster shows all managed clusters in a detached state during the following cases:
+
+- Managed clusters are created with the Hive API
+- Managed clusters connected to imported hub clusters appear in a _Pending_ state until activation data is restored, and must be manually reatteached to the new cluster
+
+The managed clusters are still connected to the original hub cluster that created the backup files.
```
Comment (sahare) [inline clusters/backup_and_restore.adoc]: This part needs to be changed from:

The resources that connect managed clusters are also called activation resources and are contained in the managed clusters backup. When you restore backups only for the _credentials_ and _resource_ backup on a new hub cluster, the new hub cluster shows all managed clusters in a detached state during the following cases:
 - Managed clusters are created with the Hive API
 - Managed clusters connected to imported hub clusters appear in a _Pending_ state until activation data is restored, and must be manually reatteached to the new cluster
The managed clusters are still connected to the original hub cluster that created the backup files.

To:

The resources that connect managed clusters are called activation resources and are contained in the managed clusters backup. When you restore backups only for the _credentials_ and _resources_ backup on a new hub, the new hub shows all managed clusters created with the Hive API in a detached state, but the managed clusters imported on the primary hub using the Import cluster operation will show only when the activation data is restored on the passive hub. At this time all the managed clusters are still connected to the original hub cluster that created the backup files.
When the activation data is restored, only managed clusters created using the Hive API will be automatically connected with the new hub. All other managed clusters appear in a _Pending_ state and must be manually reatteached to the new cluster.

**Review by sahare — CHANGES_REQUESTED** [review]
Added a comment, Thanks.

File: clusters/backup_and_restore.adoc (hunk 1 above)
Comment (dockerymick) [reply clusters/backup_and_restore.adoc]: @sahare Thanks for your feedback! I just updated the file. Let me know if I need to change anything else

File: clusters/backup_and_restore.adoc [hunk 2]
```diff
@@ -175,15 +175,12 @@ os get schedules -A | grep acm
 Resources are separately backed up in the following groups:
 
 * _Credential backup_, which contains three backup files for Hive, {product-title-short}, and user-created credentials.
```
Comment (sahare) [inline clusters/backup_and_restore.adoc]: change to _Credentials backup_

File: clusters/backup_and_restore.adoc [hunk 3]
```diff
@@ -175,15 +175,12 @@ os get schedules -A | grep acm
...
+*Note*: The _resource backup_ file contains managed cluster-specific resources, but does not contain the subset of resources that connect managed clusters to the hub cluster. The resources that connect managed clusters are called activation resources and are contained in the managed clusters backup. When you restore backups only for the _credentials_ and _resources_ backup on a new hub cluster, the new hub cluster shows all managed clusters created with the Hive API in a detached state. However, the managed clusters that are imported on the primary hub cluster using the import operation appear only when the activation data is restored on the passive hub cluster. At this time, the managed clusters are still connected to the original hub cluster that created the backup files. during the following cases:
```
Comment (sahare) [inline clusters/backup_and_restore.adoc]: - change to The `_resources backup_` file contains managed cluster-specific resources
- remove `during the following cases:` , it's not needed in this part, the next line is an independent note as:
When the activation data is restored, only managed clusters created using the Hive API are automatically connected with the new hub cluster. All other managed clusters appear in a _Pending_ state and must be manually reattached to the new cluster.

**Review by sahare — CHANGES_REQUESTED** [review]
Thanks @dockerymick , just couple more of small changes

**Review by sahare — APPROVED** [review]
Looks good, Thanks @dockerymick !

**Pull request merged**